openhands/controller/agent_controller.py

```diff
@@ -72,6 +72,8 @@
 TRAFFIC_CONTROL_REMINDER = (
     "Please click on resume button if you'd like to continue, or start a new task."
 )
+ERROR_ACTION_NOT_EXECUTED_ID = 'ERROR_ACTION_NOT_EXECUTED'
```

Comment:
```suggestion
ERROR_ACTION_NOT_EXECUTED_ID = 'AGENT_ERROR$ERROR_ACTION_NOT_EXECUTED'
```
looks like this is the current pattern